editor/src/components/user-bar.tsx

```diff
@@ -97,7 +97,7 @@ const SinglePlayerUserBar = React.memo(() => {
       <Avatar
         userPicture={userPicture}
         isLoggedIn={true}
-        size={22}
+        size={20}
```

Comment:
I would extract this 20 into a constant `AvatarSize`, it is repeated in a lot of places

Reply:
✅